### PR TITLE
P0 cart management

### PR DESCRIPTION
**Pull request by purvasharmadev**
<h2>Minor Changes</h2>
<li> Changed the links </li>
<li> Remove the background colour of product cards </li>

<h2>Main Changes </h2>
<li> Added the logout tiptool button </li>

image

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -110,6 +114,37 @@ header p {
     background-color: var(--SECONDARY-COLOR)
 }
 
+/* Logout button */
+.tooltip .tooltip-text{
+    visibility: hidden;
+    width: 7rem;
+    background-color: var(--SECONDARY-COLOR);
+    color: #fff;
```
Comment (zoef-yt) [inline index.css]: Use var for colours

File: product/product.css [hunk 2]
```diff
@@ -201,11 +201,20 @@ main h2 {
...
+    background: none;
+}
+
 .card-badge:hover{
       cursor: pointer;
   }
 
-  .card-header .category{
+  .card-footer .btn{
+      padding: 0.625rem;
```
Comment (zoef-yt) [inline product/product.css]: use px here.

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: index.css (hunk 1 above)
Comment (iRohitGaur) [reply index.css]: +1

**Review by zoyatanweer — APPROVED** [review]
good to merge after suggested changes.

**Review by iRohitGaur — APPROVED** [review]
Good to merge with updates

**Review by bossmonkey98 — APPROVED** [review]

**Pull request merged**